README.md

```diff
@@ -97,11 +97,18 @@ A pre-bundled version of VMAP-jsis available: [`vmap-js.js`](dist/vmap-js.js).
 
 You can add the script directly to your page and access the library through the `VMAP` constructor.
 
+Run `npm run dev` to watch changes.
+
+Then serve an HTML file located at the project root.
```

Comment:
I think this is not the right place for this. The goal of this part of the `README` is not to show how to parse and use an XML from a string, but simply to show how to include the script and access the library

Reply:
If we want to provide a `dev` command to debug stuff we can do it in another PR IMHO